dabest/_classes.py

```diff
@@ -1313,7 +1338,7 @@ class EffectSizeDataFrame(object):
     sizes for several comparisons."""
 
     def __init__(self, dabest, effect_size,
-                 is_paired, ci=95,
+                 is_paired, is_preceding, ci=95,
```

Comment:
Please change `preceding` to `repeated_measures` and `is_preceding` to ` repeated_measures`